server/app/views/applicant/ApplicantUpsellCreateAccountView.java

```diff
@@ -119,11 +127,39 @@ public Content render(
             messages,
             authProviderName,
             actionButtons);
-    return layout.renderWithNav(
-        request,
-        personalInfo,
-        messages,
-        createHtmlBundle(request, layout, title, bannerMessage, loginPromptModal, content),
-        applicantId);
+
+    var htmlBundle =
```

Comment:
Question (non-blocking): Do we prefer using `var` over the specific type? I don't see anything one way or the other in https://google.github.io/styleguide/javaguide.html, so just asking for my future PRs.

Reply:
When we've discussed it in the past the consensus has been you *may* use `var` if it's obvious what the type is